astro/src/components/docs/release-notes/ThemeUpdateWarning.astro

```diff
@@ -0,0 +1,11 @@
+---
+import Aside from 'src/components/Aside.astro';
+---
+
+<Aside type="theme" title="Theme update">
+<p>This version of FusionAuth includes changes to the default FusionAuth theme. If you have created custom themes in previous versions of FusionAuth, <strong>you will need to incorporate these changes into your custom themes</strong>.</p>
```

Comment:
```suggestion
<p>This version of FusionAuth includes changes to the default FusionAuth theme. If you have created custom themes in previous versions of FusionAuth, <strong class="italic">you will need to incorporate these changes into your custom themes</strong>.</p>
```
The emphasis doesn't really pop in dark mode, I think adding italic here makes it stand out a little more, though I think that `strong` is still the correct semantic tag

Reply:
For any copy style, I'd love to get Sean's opinion, and or look to the style guide for the answer.

Reply:
Sean said italics are bad for accessibility.